worker/utils/mongo.js

```diff
@@ -91,9 +91,14 @@ module.exports = {
     };
 
     const update = { $set: { startTime: new Date(), status: 'inProgress' } };
-    const options = { sort: { priority: -1, createdTime: 1 } };
-    await queueCollection.findOne({ status: 'inQueue' });
```

Comment:
Curious to know why this has been removed?

Reply:
options is still in the code base ([look at this link](https://github.com/mongodb/docs-worker-pool/pull/391/files#diff-4d2a99f21e295622e09d0249fd63dcf7d8c620ce42ebd2528c7d6384523218acR94)

Reply:
And findOne is removed because we also already have findOneAndUpdate - so the extra findOne seemed redundant to me! :)